### DISCUSSION TITLE
how to find the address of previous instruction

### DISCUSSION THREAD

**Discussion by weidehai** [x64dbg/x64dbg#3404] [Q&A]
in the call stack tab view show a table that record next address will to be executed when the called function return,so if i want to find the called function，i need to find the previous address for the next address will to be executed

**Comment by mrexodia** [comment]
Generally speaking there is no sound way to do this, but there is a heuristic implemented that can do this: `dis.prev(addr)`.

**Reply by weidehai** [reply]
nice
